README.md

```diff
@@ -143,11 +143,24 @@ There are two ECMAScript 2017 syntax changes: `async` functions, and trailing co
 
 ### What ECMAScript 2018 features do you support?
 
-Because ECMAScript 2018 is still under development, we are implementing features as they are finalized. Currently, Espree supports:
+There are seven ECMAScript 2017 syntax changes:
```

Comment:
This says 2017 but the header above says 2018. Typo?

Reply:
Ops, yes, it's typo. Thank you.